level00/README.md

```diff
@@ -0,0 +1,30 @@
+# Level00
+
+This level is intened to serve as a terse introduction to 'property-based testing'. 
+
+A significant feature of property-based testing is the randomised generation of
+inputs for a given test. 
+
+As an example, if we wanted to test a function:
```

Comment:
I find the code-block-mid-sentence thing hard to avoid but jarring. Consider:

> As an example, consider this function:
> `code block`
> If our spec says the output should be equal to the input over the range 0 to 1000 inclusive, we can express that with a property test:
> `test code block`